src/main/java/baritone/process/FarmProcess.java

```diff
@@ -60,6 +61,9 @@
     private List<BlockPos> locations;
     private int tickCount;
 
+    private int _range;
+    private Vec3i center;
```

Comment:
why are you using a vec3i instead of a blockpos? if it were a blockpos you could use ctx.playerFeet() as a default lol